e2e/test/scenarios/embedding/sdk-iframe-embedding-setup/user-settings-persistence.cy.spec.ts

```diff
@@ -29,8 +29,6 @@ describe("scenarios > embedding > sdk iframe embed setup > user settings persist
       resourceName: DASHBOARD_NAME,
     });
 
-    cy.wait("@persistSettings");
```

Comment:
There is no need for the wait anymore, as we no longer update the settings if it's selecting the same entity.